### PR TITLE
Adds debian bullseye support into citus-enterprise

### PR DESCRIPTION
**Pull request by gurkanindibay**

### CODE REVIEW

File: debian/compat [hunk 1]
```diff
@@ -1 +1 @@
-9
```
Comment (onurctirtir) [inline debian/compat:1]: why this change ?

**Review by onurctirtir — APPROVED** [review]

File: debian/compat (hunk 1 above)
Comment (gurkanindibay) [reply debian/compat:1]: https://github.com/citusdata/packaging/runs/3720189231?check_suite_focus=true#step:6:403
Gave warning in packaging process
So changed in all of them
